tests/test_configuration.py

```diff
@@ -897,3 +898,27 @@ def test_pairlist_resolving_fallback(mocker):
     assert config['pairs'] == ['ETH/BTC', 'XRP/BTC']
     assert config['exchange']['name'] == 'binance'
     assert config['datadir'] == str(Path.cwd() / "user_data/data/binance")
+
+
+@pytest.mark.parametrize("setting", [
+        ("ask_strategy", "use_sell_signal", True,
+         "experimental", "use_sell_signal", False),
+        ("ask_strategy", "sell_profit_only", False,
+         "experimental", "sell_profit_only", True),
+        ("ask_strategy", "ignore_roi_if_buy_signal", False,
+         "experimental", "ignore_roi_if_buy_signal", True),
+    ])
+def test_deprecated_experimental(mocker, default_conf, setting, caplog):
```

Comment:
i think we should also test `process_deprecated_setting()` and `check_conflicting_settings()` seperately, assuming that they we keep them once we remove experimental completely (unless you plan on removing them again ...).

Doesn't have to be existing settings we test in this case either - just something to confirm that it works.

Otherwise they'll become dead code until we change settings around again - which could be problematic in case of refactorings, since they will never be hit (neither in tests, nor in operational mode).

Reply:
Right, agree.